packages/editor/src/components/editor-history/redo.js

```diff
@@ -24,14 +27,7 @@ export default compose( [
 	withSelect( ( select ) => ( {
 		hasRedo: select( 'core/editor' ).hasEditorRedo(),
 	} ) ),
-	withDispatch( ( dispatch, ownProps ) => ( {
-		redo: () => {
-			// If there are no redo levels this is a no-op, because we don't actually
-			// disable the button.
-			// See: https://github.com/WordPress/gutenberg/issues/3486
-			if ( ownProps.hasRedo ) {
-				dispatch( 'core/editor' ).redo();
-			}
-		},
+	withDispatch( ( dispatch ) => ( {
+		redo: () => dispatch( 'core/editor' ).redo(),
```

Comment:
nitpick: can we simplify to ```redo: dispatch( 'core/editor' ).redo,```?

Reply:
It was like that before I made the change, though I'm not immediately sure why. I'll give it a shot.